framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -298,7 +296,7 @@ private void clampToLowerBound(InferenceResult fromArgSupertypes, List<Annotated
             InferredValue inferred = fromArgSupertypes.get(targetDecl.getUnderlyingType());
             if (inferred != null && inferred instanceof InferredType) {
                 final AnnotatedTypeMirror lowerBoundAsArgument =
-                     AnnotatedTypes.asSuper(types, typeFactory, targetDecl.getLowerBound(), ((InferredType) inferred).type);
+                        AnnotatedTypes.asSuper(types, typeFactory, targetDecl.getLowerBound(), ((InferredType) inferred).type);
```

Comment:
This follows the Java convention.  I am not a particular fan of this convention but the previous lines also follow it.  So this will make it consistent.